### PR TITLE
(FACT-2419) Module structure that respects rubocop

### PR DESCRIPTION
**Pull request by oanatmaria**

### CODE REVIEW

File: spec/custom_facts/util/confine_spec.rb [hunk 1]
```diff
@@ -77,7 +77,7 @@ def confined(fact_value, *confines)
       expect(confined([3, 4], [3, 4])).to be true
     end
 
-    it "returns true if any of the provided symbol values matches the fact's string value" do
+    it "returns true if any of the provided symbol values matches the fact's value as a sctring" do
```
Comment (Filipovici-Andrei) [inline spec/custom_facts/util/confine_spec.rb]: probably string instead of sctring?

**Review by Filipovici-Andrei — CHANGES_REQUESTED** [review]

File: lib/facts/aix/ssh.rb [hunk 2]
```diff
@@ -16,7 +16,7 @@ def fact_value
       end
 
       def resolver_data
-        Resolvers::SshResolver.resolve(:ssh)
+        Facter::Resolvers::SshResolver.resolve(:ssh)
```
[the remark this answers is not in the document]
Comment (oanatmaria) [reply lib/facts/aix/ssh.rb]: Resolvers are not in the scope of this PR and they will be tracked under another ticket

File: lib/facts/macosx/network_ip.rb [hunk 3]
```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
-module Facter
+module Facts
   module Macosx
-    class NetworkIP
+    class NetworkIp
```
[the remark this answers is not in the document]
Comment (oanatmaria) [reply lib/facts/macosx/network_ip.rb]: this file should and will be deleted

File: lib/facts/macosx/ruby/sitedir.rb [hunk 4]
```diff
@@ -1,14 +1,16 @@
 # frozen_string_literal: true
 
-module Facter
+module Facts
   module Macosx
-    class RubySitedir
-      FACT_NAME = 'ruby.sitedir'
+    module Ruby
+      class Sitedir
```
[the remark this answers is not in the document]
Comment (oanatmaria) [reply lib/facts/macosx/ruby/sitedir.rb]: the file is sitedir, in order to be with a capital D the file name should have been site_dir

**Comment by oanatmaria** [comment]
Everything passed on puppet-agent https://jenkins-master-prod-1.delivery.puppetlabs.net/view/Adhoc/job/platform_puppet-agent-extra_puppet-agent-integration-suite_adhoc-ad_hoc/763/. Also facter's acceptance tests.

**Review by Filipovici-Andrei — APPROVED** [review]

File: .rubocop_todo.yml [hunk 5]
```diff
@@ -119,354 +119,8 @@ RSpec/FilePath:
...
-    - 'spec/facter/facts/solaris/os/release_spec.rb'
-    - 'spec/facter/facts/solaris/path_spec.rb'
-    - 'spec/facter/facts/solaris/ruby/platform_spec.rb'
-    - 'spec/facter/facts/solaris/ruby/sitedir_spec.rb'
-    - 'spec/facter/facts/solaris/ruby/version_spec.rb'
-    - 'spec/facter/facts/solaris/solaris_zones/current_spec.rb'
-    - 'spec/facter/facts/solaris/solaris_zones/zone_spec.rb'
-    - 'spec/facter/facts/solaris/zfs_featurenumbers_spec.rb'
-    - 'spec/facter/facts/solaris/zfs_version_spec.rb'
     - 'spec/facter/facts/solaris/zpool_featureversions_spec.rb'
```
Comment (gimmyxd) [inline .rubocop_todo.yml]: This should also be fixd:
```❯ cat spec/facter/facts/solaris/zpool_featureversions_spec.rb
# frozen_string_literal: true

describe Facts::Solaris::ZpoolFeaturenumbers do
  describe '#call_the_resolver' do
    subject(:fact) { Facts::Solaris:
....
```

File: .rubocop_todo.yml [hunk 6]
```diff
@@ -119,354 +119,8 @@ RSpec/FilePath:
...
-    - 'spec/facter/facts/macosx/identity/privileged_spec.rb'
-    - 'spec/facter/facts/macosx/identity/uid_spec.rb'
-    - 'spec/facter/facts/macosx/identity/user_spec.rb'
-    - 'spec/facter/facts/macosx/is_virtual_spec.rb'
-    - 'spec/facter/facts/macosx/isa_spec.rb'
-    - 'spec/facter/facts/macosx/kernel_spec.rb'
-    - 'spec/facter/facts/macosx/kernelmajversion_spec.rb'
-    - 'spec/facter/facts/macosx/kernelrelease_spec.rb'
-    - 'spec/facter/facts/macosx/kernelversion_spec.rb'
     - 'spec/facter/facts/macosx/load_average_spec.rb'
```
Comment (gimmyxd) [inline .rubocop_todo.yml]: the files should be called: `load_averages_spec.rb`

**Pull request merged**